pkg/pillar/iptables/ssh.go

```diff
@@ -95,3 +100,96 @@ func dropPortRange(startPort int, endPort int) {
 	IptableCmd("-A", "INPUT", "-p", "tcp", "--dport", portStr, "-j", "REJECT", "--reject-with", "tcp-reset")
 	Ip6tableCmd("-A", "INPUT", "-p", "tcp", "--dport", portStr, "-j", "REJECT", "--reject-with", "tcp-reset")
 }
+
+// With flow monitoring happening, any unmarked connections:
+// 1) Not matching any of the INPUT ACL in PREROUTING chain
+// 2) Not initiated by applications
+// will be dropped (sent out of dummy interface). But, we still
+// want some control protocols running on dom0 to run. We mark such
+// connections with markings from reserved space and let the ACLs
+// in INPUT chain make the ACCEPT/DROP/REJECT decisions.
+func markControlFlows() {
+	ControlProtocolMarkingIDMap = map[string]string{
```

Comment:
Can't this map be defined once as a variable where it is defined? Or even better, a const map?

Reply:
Will see how that can be done.